controllers/scriptStorage.js

```diff
@@ -429,6 +443,8 @@ exports.storeScript = function (aUser, aMeta, aBuf, aCallback, aUpdate) {
       }
 
       aScript.save(function (aErr, aScript) {
+        // WARNING: No error handling
```

Comment:
btw notice this! I ran into an error server side and had no idea where the error was coming from... might be nice to handle this at some point.